sequence/callbacks.py

```diff
@@ -76,9 +76,14 @@ def callback(**kwargs):
                 packed_padded, padded = ds.get_batch(
                     i * 100, i * 100 + 100, device=device
                 )
+                if model.__class__.__name__ == "LSTM":
```

Comment:
You could create an args object here. This way you only do the check ` if model.__class__.__name__ == "LSTM"` once.

Something like:
```python
args = [packed_padded]
if model.__class__.__name__ == "LSTM":
    # could also be directly appendedn, however this is more readable
    state_h, state_c = model.init_state(padded.shape[1])
    args.append((state_h, state_c))

with torch.no_grad():
    pred = model(*args)
```